draft-ietf-rtcweb-security-arch.xml

```diff
@@ -1376,11 +1376,18 @@ extension-att-value = 1*(%x01-09 / %x0b-0c / %x0e-3a / %x3c-ff)
               </t>
               <figure>
                 <artwork><![CDATA[
-  https://example.com/.well-known/idp-proxy/example
+  https://identity.example.com/.well-known/idp-proxy/example
   ]]></artwork>
               </figure>
               <t>
-                The IdP MAY redirect requests to this URL, but they MUST retain
+                The IdP MAY redirect requests to:
+               </t>
+               <figure>
+                <artwork><![CDATA[
+  https://example.com/.well-known/idp-proxy/example
```

Comment:
This implies that you can only redirect a) by erasing domain name labels, and b) to a `.well-known` URI.  Neither is true.

I don't think that you need to make this change, the example tweak is OK.  (Or you could include an example for something completely different, like `https://idp.example/foo/`)